dataland-frontend/src/components/forms/CreateSmeDataset.vue

```diff
@@ -0,0 +1,283 @@
+        </SubmitSideBar>
+      </div>
+    </template>
+  </Card>
+</template>
+<script lang="ts">
+import { FormKit } from "@formkit/vue";
+import { computed, defineComponent, inject } from "vue";
+import { assertDefined } from "@/utils/TypeScriptUtils";
+import { useRoute } from "vue-router";
+import { checkCustomInputs, checkIfAllUploadedReportsAreReferencedInDataModel } from "@/utils/ValidationsUtils";
+import UploadReports from "@/components/forms/parts/UploadReports.vue";
+import { smoothScroll } from "@/utils/SmoothScroll";
+import { createSubcategoryVisibilityMap } from "@/utils/UploadFormUtils";
+import { ApiClientProvider } from "@/services/ApiClients";
+import Card from "primevue/card";
+import Calendar from "primevue/calendar";
+import type Keycloak from "keycloak-js";
+import PrimeButton from "primevue/button";
+import { type Category, type Subcategory } from "@/utils/GenericFrameworkTypes";
+import { AxiosError } from "axios";
+import { type CompanyAssociatedDataSmeData, type CompanyReport, DataTypeEnum } from "@clients/backend";
+import { smeDataModel } from "@/frameworks/sme/UploadConfig";
+import UploadFormHeader from "@/components/forms/parts/elements/basic/UploadFormHeader.vue";
+import YesNoFormField from "@/components/forms/parts/fields/YesNoFormField.vue";
+import NumberFormField from "@/components/forms/parts/fields/NumberFormField.vue";
+import MultiSelectFormField from "@/components/forms/parts/fields/MultiSelectFormField.vue";
+import SubmitButton from "@/components/forms/parts/SubmitButton.vue";
+import SubmitSideBar from "@/components/forms/parts/SubmitSideBar.vue";
+import SuccessMessage from "@/components/messages/SuccessMessage.vue";
+import FailMessage from "@/components/messages/FailMessage.vue";
+import DateFormField from "@/components/forms/parts/fields/DateFormField.vue";
+import SingleSelectFormField from "@/components/forms/parts/fields/SingleSelectFormField.vue";
+import BigDecimalExtendedDataPointFormField from "@/components/forms/parts/fields/BigDecimalExtendedDataPointFormField.vue";
+import NaceCodeFormField from "@/components/forms/parts/fields/NaceCodeFormField.vue";
+import { type DocumentToUpload, uploadFiles } from "@/utils/FileUploadUtils";
+import { type ObjectType } from "@/utils/UpdateObjectUtils";
+import { formatAxiosErrorMessage } from "@/utils/AxiosErrorMessageFormatter";
+import { getFilledKpis } from "@/utils/DataPoint";
+
+export default defineComponent({
+  setup() {
+    return {
+      getKeycloakPromise: inject<() => Promise<Keycloak>>("getKeycloakPromise"),
+    };
+  },
+  name: "CreateSmeDataset",
+  components: {
+    FormKit,
+    UploadFormHeader,
+    MultiSelectFormField,
+    NumberFormField,
+    Card,
+    PrimeButton,
+    Calendar,
+    SuccessMessage,
+    FailMessage,
+    SubmitButton,
+    SubmitSideBar,
+    DateFormField,
+    SingleSelectFormField,
+    BigDecimalExtendedDataPointFormField,
+    YesNoFormField,
+    NaceCodeFormField,
+    UploadReports,
+  },
```

Comment:
I did not check out the frontend yet but we have the tooltip set in the directives block for many other frameworks here.
I don't know the usage of it, but are the tooltips maybe not displayed correctly on the upload page?

Reply:
Die Tooltips sehen ganz normal aus meines Erachtens.
Sie landen über ` :description="field.description"` im `<component>` tag in den upload-form-elementen.

Kann es sein dass der directives Block gar nicht notwendig ist?

Reply:
Wenn wir nichts vermissen, lassen wir ihn weg. Ob er notwendig ist finden wir hoffentlich zeitnah heraus.